setup.py

```diff
@@ -32,9 +32,6 @@
     'install': [
         'cf-units',
         'dask[array]',
-        # fiona: 1.8.18/py39, they seem weary to build manylinux wheels
-        # so we may have to install from conda-forge in the future
-        'fiona',
```

Comment:
The list in setup.py lists all dependencies, regardless of how they're installed. Could you put fiona back her @valeriupredoi? If it's installed by conda already, pip won't try to install it.

Reply:
aha will do now!

Reply:
https://github.com/ESMValGroup/ESMValCore/pull/990 pls approve so we merge and release the fiona baby

Reply:
Thanks!

Reply:
cheers for pointing out and reviewing, bud!